### PR TITLE
Upgrade Pliny to latest version (0.16.3)

### PR DESCRIPTION
**Pull request by dmcinnes**
Mainly for this this and other fixes: Updated instrumentation handling https://github.com/interagent/pliny/pull/218

### CODE REVIEW

File: config/initializers/rollbar.rb [hunk 1]
```diff
@@ -3,9 +3,11 @@
     config.enabled = (Config.rack_env == 'production')
     config.environment = Config.console_banner || 'production'
     config.access_token = ENV["ROLLBAR_ACCESS_TOKEN"]
+    config.logger = Pliny::RollbarLogger.new
     config.use_sucker_punch
```
Comment (mikehale) [inline config/initializers/rollbar.rb]: should we be using both sucker_punch and sidekiq?

**Comment by mikehale** [comment]
Just the 1 comment, but overall LGTM.

File: config/initializers/rollbar.rb (hunk 1 above)
Comment (dmcinnes) [inline config/initializers/rollbar.rb]: Ah missed that! Sidekiq comes in default for Pliny. I'll leave it sucker_punch if that seems to work.

**Comment by mikehale** [comment]
:+1:

**Comment by dmcinnes** [comment]
Thanks!

**Pull request merged**